src/Input/Input.js

```diff
@@ -68,6 +68,14 @@ export const styleSheet = createStyleSheet('MuiInput', (theme) => {
         appearance: 'none',
       },
     },
+    multiline: {
```

Comment:
I have tried with the following properties, that looks all good to me, we can kill `multiline-wrapper`.
```js
    multiline: {
      resize: 'none',
      padding: '0 0 6px',
      marginTop: 6,
    },
    'multiline-wrapper': {
    }
```

![capture d ecran 2017-04-18 a 20 28 12](https://cloud.githubusercontent.com/assets/3165635/25146806/927670f4-2475-11e7-9226-1d31694b566d.png)